controllers/user.js

```diff
@@ -22,8 +22,17 @@ const deleteUser = async (req, res) => {
   res.status(204).end()
 }
 
+const updateUser = async (req, res) => {
+  const { id } = req.params
+  const body = req.body
```

Comment:
please think how to solve the problem with fields that are not meant to be updated by this endpoint, gotta have some validation in service or something to filter out fields that can be updated
+
atm this function in the service is meant to update some fields inside the app, without human access from the outside, so we are sure that only right fields will be updated.
consider that those fields must be available for internal update, and not avail for external

Reply:
so mby 2 update functions inside the service or IFs, i dunnu

atm it is dangerous to leave it as it is, coz lets say user role can be changed from the outside

Reply:
you mean make some check if fields from body contains in User model schema inside service before updating them?

Reply:
added filtering